content/blog/2021/announcing-pycon-india.md

```diff
@@ -0,0 +1,43 @@
+Title: PyCon India is back online for 2021
+Date: 2021-02-25 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2021
+Slug: announce-pycon-india-2021
+Summary: Announcing the return of PyCon India as an online conference for 2021.
+
+We hope you've had a safe and healthy year. 
+
+We are very excited to announce that [**PyCon India**](https://inpycon21.netlify.app/) is back online for 2021.
+
+Get ready for 4 days of non-stop fun and learning with 4 conference tracks, sessions, BOFs, community booths, workshops, and devsprints.
+
+Accessing the situation with COVID-19 and the rate of vaccination, we have decided to keep PyCon India 2021 as an online event to ensure the safety of everyone involved. We know you all are excited to meet everyone in person and celebrate Python in the "real world", but everyone's health and safety are the utmost priority at this time. 
```

Comment:
```suggestion
Assessing the situation with COVID-19 and the rate of vaccination, we have decided to keep PyCon India 2021 as an online event to ensure the safety of everyone involved. We know folks were excited to meet everyone in person and celebrate Python in the "real world" by having an offline PyCon India 2021. We do too but maintaining everyone's health and safety is our utmost priority at all times. 
```

Reply:
"We were too,"?

And how about removing "at all times". Just keep ...health is our utmost priority?